website/docs/docs/fusion/supported-features.md

```diff
@@ -87,7 +87,7 @@ Here are some additional considerations if using the Fusion CLI without the VS C
 
 ## Limitations
 
-If your project is using any of the features listed in the following table, you can use Fusion, but you won't be able to fully migrate all your workloads because you have:
+If your project is using any of the features or external tooling listed in the following table, you can use Fusion, but you won't be able to fully migrate all your workloads because you have:
```

Comment:
```suggestion
If your project is using any of the features listed in the following table, you can use Fusion, but you won't be able to fully migrate all your workloads because you have:
```